src/officehours_ui/static/css/main.css

```diff
@@ -641,3 +641,7 @@ div[role=main] {
 .meeting-type-message {
 	padding-left: 5px
 }
+
+.queue-list-badge {
+	font-size: 90%;
+}
```

Comment:
Noticed a few missing LFs at EOF.

Reply:
So you just wanted a return at the end of the file? Did that.

Reply:
Oh, there are a couple others.